tungsten_ci_utils/nexus_cleanup/clean_up.groovy

```diff
@@ -0,0 +1,117 @@
+import org.sonatype.nexus.repository.storage.StorageFacet;
+import org.sonatype.nexus.common.app.GlobalComponentLookupHelper
+import org.sonatype.nexus.repository.maintenance.MaintenanceService
+import org.sonatype.nexus.repository.storage.ComponentMaintenance
+import org.sonatype.nexus.repository.storage.Query;
+import org.sonatype.nexus.script.plugin.RepositoryApi
+import org.sonatype.nexus.script.plugin.internal.provisioning.RepositoryApiImpl
+import com.google.common.collect.ImmutableList
+import org.joda.time.DateTime;
+import org.slf4j.Logger
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+def retentionCount = 15;
+def retentionDays = 30;
+def repositoryName = 'BartsDockerRepo';
+def whitelistCore = ["latest", "ocata", "newton", "queens"].toArray();
+def whitelist = ["latest","40", "94", "122", "129", "161", "214", "309", "360"].toArray();
```

Comment:
These are suffixes, don't see another usage. So rename to e.g. whitelisted_tag_suffixes